src/rollingBall.js

```diff
@@ -23,8 +23,8 @@ import min from 'ml-array-min';
  *     https://cran.r-project.org/web/packages/baseline/index.html
  * @export
  * @param {Array} spectrum
- * @param {Number} [options.windowM]: width of local window for minimization/maximization, defaults to 4% of the spectrum length
- * @param {Number} [options.windowS]: width of local window for smoothing, defaults to 8% of the specturm length
+ * @param {Number} [options.windowM] - width of local window for minimization/maximization, defaults to 4% of the spectrum length
```

Comment:
Please add before
* @param {object} [options={}]

Reply:
thanks for spotting!